remark-plugins/extract-image-class.js

```diff
@@ -0,0 +1,26 @@
+import { visit } from 'unist-util-visit'
+export function extractImageClass() {
```

Comment:
I'm consciously trying to avoid default exports, since they generally have worse ergonomics than named exports:

https://basarat.gitbook.io/typescript/main-1/defaultisbad#dynamic-imports

That said, I'm happy to change it if they're the convention here!

Reply:
I'm fine with either. Not sure what the "standard" is for remark plugins specifically, but I'd probably follow what other plugins do so we have a similar DX

Reply:
Some very valid reasons in that link imo.
Other plugins seem to mostly use default exports tho :/ but many of them don't use esm either so idk how much of an inspiration they should be for us.